test/images/dnsutils/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=alpine
```

Comment:
@tianon ok. Thank you very much for responding. So armhf is deprecated in favor for arm32v6?
Is there any arm32v7?